### DISCUSSION TITLE
Unable to open px-submission tool

### DISCUSSION THREAD

**Discussion by sarahkelliher** [PRIDE-Archive/pride-resources#38] [PRIDE Submission Issues] [answered]
Hi there, 

I am unable to open the px submission tool (trying version 2.7.3). I am using Windows 10 and have latest version of Java installed. I have tried the suggestions in this previous discussion:  https://github.com/orgs/PRIDE-Archive/discussions/2

However this did not resolve the issue. 

<img width="513" alt="image" src="https://github.com/PRIDE-Archive/pride-resources/assets/143716285/d8405323-2de0-41f6-8f45-388473d5d993">

**Comment by selvaebi** [comment]
As per discussion after unzipping ,

cd <path/to/the/px-submission-tool/directory> Navigate to your submission tool folder

java -jar px-submission-tool-.jar to open the tool. Please replace the version number.

**Reply by sarahkelliher** [reply]
Thanks for your reply. Unfortunately that did not work- I have unzipped the folder and it is on my desktop. I have attached a screenshot of the error.

<img width="812" alt="image" src="https://github.com/PRIDE-Archive/pride-resources/assets/143716285/854a26b4-62ba-4f50-8b4b-1aabc7aed1fb">

**Reply by ypriverol** [reply, marked as answer]
@sarahkelliher you are in the same folder `C:\Users\sarah` you have to move to the folder of the submission tool I guess the previos command to the execution is: 

`cd {folder of the submission tool}`

**Reply by sarahkelliher** [reply]
Thank you this worked!
